### DISCUSSION TITLE
Is there a good purpose for the locale that is specified?

### DISCUSSION THREAD

**Discussion by joro75** [DurgNomis-drol/mytoyota#118] [General]
@DurgNomis-drol 
Do you know if there is a real difference if a different locale is being specified during the creation of the MyT instance?

I always used 'en-gb' for the locale, and just didn't try to use a different locale. What happens if we for example specify 'fr-fr' as the locale? I do not expect that the keys in the JSON response will change on this.

Some possibilities:
* Are the error message changing, such that the description of the returned error is suddenly translated in the local language?
* Is the numbering format of the returned numbers changed to use , instead of . ?
* Is the date formatting changed when historical data is returned or requested?
* I do know that the values are not suddenly returned in miles or km, based on the selected locale?

If it doesn't matter which locale is selected, wouldn't it just be easier to specify a locale that is always the same?

**Comment by DurgNomis-drol** [comment]
As fare as i know, it does not have any effect. I have tested with `da-dk` and nothing changed for me. Right now my Home Assistant integration is hard coded to `en-gb` and it still returns it's data in metric.I believe that the locale has to be valid one in Europe, that is the only limitation i have found so far. 

My reason for not hard coding it when i made this library, was not i wasn't sure about what it did and didn't. We can change it to default to a standard one instead of raising an error as we do now.

**Comment by joro75** [comment]
Ok, that confirms the idea that I had. I also tried to use the mytoyota library while passing it another locale (`fr-fr`). I can confirm that there is no visible difference. 

I also do agree that we should keep the locale as an optional argument, so it is possible to change it, if we someone finds that it is necessary in the future. Do you agree to provide a default value to the `locale` argument of `MyT`? Shall we just use `da-dk`, to prevent that it suddenly changes to an imperial system if `en-gb` is chosen as the default? 

I will also modify my own code (Domoticz-Toyota-Plugin) to hide the option for the end-user to specify a locale. It is just not needed.

**Reply by DurgNomis-drol** [reply]
The default in my Home Assistant integration is `en-gb`, but you are right it is probably a good idea to move it to a country that uses metrics. Let's just use `da-dk` or `de-de` that's is your decision. 😃

**Reply by joro75** [reply]
Ok. Lastest code is now using `da-dk` as the default locale, and it will use `europe` as the default region.
